src/components/Calculator.js

```diff
@@ -0,0 +1,33 @@
+const ResultArea = () => <div type="text" className="resultArea"> 0 </div>;
+const CalcButtons = () => (
+  <div className="calcContainer">
+    <button type="button" className="calcItem btn-number">AC</button>
+    <button type="button" className="calcItem btn-number">+/-</button>
+    <button type="button" className="calcItem btn-number">%</button>
+    <button type="button" className="calcItem btn-sign">/</button>
+    <button type="button" className="calcItem btn-number">7</button>
+    <button type="button" className="calcItem btn-number">8</button>
+    <button type="button" className="calcItem btn-number">9</button>
+    <button type="button" className="calcItem btn-sign">X</button>
+    <button type="button" className="calcItem btn-number">4</button>
+    <button type="button" className="calcItem btn-number">5</button>
+    <button type="button" className="calcItem btn-number">6</button>
+    <button type="button" className="calcItem btn-sign">-</button>
+    <button type="button" className="calcItem btn-number">1</button>
+    <button type="button" className="calcItem btn-number">2</button>
+    <button type="button" className="calcItem btn-number">3</button>
+    <button type="button" className="calcItem btn-sign">+</button>
+    <button type="button" className="calcItem btn-number btn-zero">0</button>
+    <button type="button" className="calcItem btn-number">.</button>
+    <button type="button" className="calcItem btn-sign"> = </button>
+  </div>
+);
```

Comment:
- [ ] **[OPTIONAL]** Kindly move these components to new files. Components are usually placed in their files in React.